doc/GuideForCommitters.md

```diff
@@ -93,19 +93,19 @@ build, add a comment to the pull request that follows the syntax:
 Where `{platform-list}` is a comma-separated list of platforms chosen from the
 following tags:
 
-| Tag      | Platform |
-| :----    | :------- |
-| zlinux   | 64-bit Linux on Z |
-| zos      | 64-bit zOS |
-| plinux   | 64-bit Power Linux LE |
-| aix      | 64-bit AIX |
-| x32linux | 32-bit Linux |
-| xlinux   | 64-bit Linux |
-| xcrlinux | 64-bit Compressed References Linux |
-| win      | 64-bit Windows |
-| osx      | 64-bit macOS |
-| arm      | 32-bit AArch32 (cross-compile build only) |
-| aarch64  | 64-bit AArch64 (cross-compile build only) |
+| Tag      | Platform                                  | Alias               |
+| :----    | :-------                                  | :----               |
+| zlinux   | 64-bit Linux on Z                         | linux_390-64        |
+| zos      | 64-bit zOS                                | zos_390-64          |
+| plinux   | 64-bit Power Linux LE                     | linux_ppc-64_le_gcc |
+| aix      | 64-bit AIX                                | aix_ppc-64          |
+| x32linux | 32-bit Linux                              | linux_x86           |
+| xlinux   | 64-bit Linux                              | linux_x86-64        |
+| win      | 64-bit Windows                            | win_x86-64          |
+| osx      | 64-bit macOS                              | osx_x86-64          |
+| arm      | 32-bit AArch32 (cross-compile build only) | linux_arm           |
+| aarch64  | 64-bit AArch64 (cross-compile build only) | linux_aarch64       |
+| riscv    | 64-bit RISC-V (cross-compile build only)  | linux_riscv64_cross |
```

Comment:
These should say "Linux" like xlinux, zlinux, etc. do.

Reply:
Updated.